generators/spring-service/prompts.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
Can we reuse the prompts from the base ? By that we need not worry to maintain this file?

Reply:
Do you already have any suggestion on how to do that?

Reply:
instead of linking to this prompts file link the base prompts file

Reply:
something like this https://github.com/jhipster/jhipster-kotlin/blob/a3e591059b89048d5c6a9c560d03bd5578e48b60/generators/server/index.js#L46

Reply:
still `super._prompting()` is not used right?